test/voicefocus/VoiceFocusDeviceTransformer.test.ts

```diff
@@ -232,6 +232,26 @@ describe('VoiceFocusDeviceTransformer', () => {
           supported: true,
         };
 
+        it('can be destroyed', async () => {
+          const spec: VoiceFocusSpec = {
+            variant: 'c10',
+            simd: 'force',
+          };
+          const logger = new MockLogger();
+          const transformer = await VoiceFocusDeviceTransformer.create(
+            spec,
+            {
+              preload: false,
+              logger,
+            },
+            configWithURL
+          );          
+          await VoiceFocusDeviceTransformer.destroyVoiceFocus(transformer);
+          expect(transformer.isSupported()).to.be.true;
+          expect(transformer).to.not.be.undefined;
```

Comment:
How do these two `expect` calls verify that the transformer has been destroyed?

Reply:
fixed